processingchain/src/test/groovy/org/orbisgis/orbisprocess/geoclimate/processingchain/GeoIndicatorsChainTest.groovy

```diff
@@ -149,6 +150,7 @@ class GeoIndicatorsChainTest {
     }
 
     @Test
+    @Disabled
```

Comment:
This test doesn't work any more ?

Reply:
The test was disabled on the dev branch.